skins/complifi.scss

```
@@ -0,0 +1 @@
+
```

Please type your code in the `.scss` file.
```suggestion
.complifi {
  --primary-color: #4aa1ff;
  --bg-color: #fdfdfd;
  --text-color: #0c1b32;
  --link-color: #0c1b32;
  --heading-color: #20150;
  --border-color: #e0e0e0;
  --header-bg: #ededf1;
  --block-bg: #f5f5f7;
}

@media (prefers-color-scheme: dark) {
  .complifi {
    --primary-color: #4aa1ff;
    --bg-color: #05142c;
    --text-color: #cccccc;
    --link-color: #ffffff;
    --heading-color: #ffffff;
    --border-color: #2d3a4e;
    --header-bg: #0c1b32;
    --block-bg: #0c1b32;
  }
}
```